apps/taxes/src/modules/avatax/avatax-config.ts

```diff
@@ -31,13 +53,18 @@ export const avataxInstanceConfigSchema = z.object({
 
 export type AvataxInstanceConfig = z.infer<typeof avataxInstanceConfigSchema>;
 
-export const obfuscateAvataxConfig = (config: AvataxConfig) => ({
+export const obfuscateAvataxConfig = (config: AvataxConfig): AvataxConfig => ({
   ...config,
-  username: obfuscateSecret(config.username),
-  password: obfuscateSecret(config.password),
+  credentials: {
+    ...config.credentials,
+    username: obfuscateSecret(config.credentials.username),
```

Comment:
username should be obfuscated?

Reply:
But are you sure this is a secret? Its strange, I have never seen to obfuscate any name/nickname type of field